docs/src/markdown/extensions/tasklist.md

```diff
@@ -141,9 +155,10 @@ Classes               | Description
 
 ## Options
 
-Option            | Type | Default      | Description
------------------ | ---- | ------------ | ------------
-`custom_checkbox` | bool | `#!py3 False` | Generate task lists in such a way as to allow for styling the check box with CSS.
+| Option               | Type | Default       | Description                                                                       |
```

Comment:
Please don't rewrite table syntax that I am using.  Please remove leading and trailing pipes (`|`).  I understand some may prefer to have tables flanked by those, but I do not.